### ISSUE TITLE
[BUG] getAllPits no get handler for PIT/_all

### ISSUE THREAD

**Issue by b5414** [opensearch-project/opensearch-js#403]
### What is the bug?
`getAllPits();` is not working
err: `no handler found for uri [/_search/point_in_time/_all] and method [GET]`

### How can one reproduce the bug?
1. Copy [this code](https://github.com/opensearch-project/opensearch-js/blob/main/USER_GUIDE.md#using-aws-v3-sdk)
2. Combine with [this one](https://github.com/opensearch-project/opensearch-js/blob/main/USER_GUIDE.md#get-all-pits)

```js
try{
    var response = await client.getAllPits();
    console.log(response.body);
}catch(e){
    console.log(e); // Photo below
}
```

### What is the expected behavior?
Get all PITs 🙂

### What is your host/environment?
Windows 10 pro latest

### Do you have any screenshots?
[Photo](https://imgur.com/GS09ysR)

### Do you have any additional context?
I don't have any PITs

```json
"@aws-sdk/client-opensearch": "^3.278.0",
"@aws-sdk/credential-provider-node": "^3.278.0",
"@opensearch-project/opensearch": "^2.2.0",
"aws-sdk": "^2.1322.0",
"axios": "^1.3.4",
```

**Comment by nhtruong** [comment]
@b5414 Which version of OpenSearch are you on?

**Comment by b5414** [comment]
Aws: `OpenSearch 2.3 (latest)`
Help says: `v2.3.0`

**Comment by nhtruong** [comment]
@b5414 the new Point-in-time feature is only available in OpenSearch 2.4 onward.
